Framework/ICat/test/CompositeCatalogTest.h

```diff
@@ -177,7 +177,7 @@ class CompositeCatalogTest : public CxxTest::TestSuite {
    * Create a compositeCatalog and add two DummyCatalogs to it.
    * @return A shared pointer to a CompositeCatalog.
    */
-  const std::shared_ptr<CompositeCatalog> createCompositeCatalog() {
+  const std::shared_ptr<CompositeCatalog> &createCompositeCatalog() {
```

Comment:
As this is a creation method then I would expect we should just return the pointer and not a reference.